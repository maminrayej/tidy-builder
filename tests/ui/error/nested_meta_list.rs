#[derive(tidy_builder::Builder)]
struct MyStruct {
    #[builder(builder(each = "each"))]
    args: Vec<String>,
}

fn main() {}
