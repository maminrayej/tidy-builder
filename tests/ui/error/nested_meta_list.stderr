error: Nested attributes are not supported
 --> tests/ui/error/nested_meta_list.rs:3:15
  |
3 |     #[builder(builder(each = "each"))]
  |               ^^^^^^^^^^^^^^^^^^^^^^
